ngraph/test/backend/gather_nd.in.cpp

```diff
@@ -479,3 +480,456 @@ NGRAPH_TEST(${BACKEND_NAME}, gather_nd_batch_dims2_lead_dims) {
     EXPECT_TRUE(
         test::all_close_f((vector<float>{2, 5, 11, 13, 19, 23}), read_vector<float>(result), MIN_FLOAT_TOLERANCE_BITS));
 }
+
+//-----v8-----
+NGRAPH_TEST(${BACKEND_NAME}, gather_nd_v8_single_indices) {
```

Comment:
These tests are frozen. Please move these tests to new infrastructure:
https://github.com/openvinotoolkit/openvino/tree/master/docs/template_plugin/tests/functional/op_reference